hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/window/components/SimpleGroupsLayout.java

```diff
@@ -244,12 +260,37 @@ private RadioButtonGroup<ERROR_THRESHOLD_OPTIONS> createErrorThresholdOptionGrou
         return errorThresholdOptions;
     }
 
-    private GridLayout buildLayout() {
+    public CheckBox createConfirmationToggle() {
+        final CheckBox confirmationToggle = FormComponentBuilder.createCheckBox(
+                UIComponentIdProvider.ROLLOUT_CONFIRMATION_REQUIRED, binder,
+                ProxySimpleRolloutGroupsDefinition::isConfirmationRequired,
+                ProxySimpleRolloutGroupsDefinition::setConfirmationRequired);
+        confirmationToggle.addStyleName(ValoTheme.CHECKBOX_SMALL);
+
+        return confirmationToggle;
+    }
+
+    public Link createConfirmationHelpLink(final UiProperties uiProperties) {
+        final String confirmationFlowHelpUrl = uiProperties.getLinks().getDocumentation().getConfirmationFlow();
+        final Link confirmationHelpLink = new Link("", new ExternalResource(confirmationFlowHelpUrl));
+
+        confirmationHelpLink.setTargetName("_blank");
+        confirmationHelpLink.setIcon(VaadinIcons.QUESTION_CIRCLE);
+        confirmationHelpLink.setDescription(i18n.getMessage("tooltip.documentation.link"));
+
+        return confirmationHelpLink;
+    }
+
+    private GridLayout buildLayout(final boolean isConsentFlowActive) {
         final GridLayout gridLayout = new GridLayout();
         gridLayout.setMargin(false);
         gridLayout.setSpacing(true);
         gridLayout.setSizeUndefined();
-        gridLayout.setRows(4);
+        if (isConsentFlowActive) {
```

Comment:
Have also mentioned it in the manual assignment PR, if we could avoid usage of "flow", also here as well, we could just say isUserConsentEnabled() or even something better coz this is a UI class?